client/src/components/userCard.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <v-card color="title" class="py-10 ma-5 hover-card">
+    <div class="d-flex align-center justify-center items-center py-0 font-weight-bold">
+      <profileImage  v-show="user" :image="user?.image" :fullName="user?.full_name"/>
```

Comment:
i think passing the whole user object will be better

```suggestion
      <profileImage  v-show="user" :user="user" />
```